README.md

```diff
@@ -4,6 +4,28 @@
 
 This repository contains the source for the Google-maintained Tomcat [docker](https://docker.com) image.
 
+# Using the Tomcat image
+This image does not have official release, so you will need to build it first.
+
+See [DEVELOPING.md](DEVELOPING.md)
+
+You will need to create a Dockerfile based on the current image and add your application as a war file.
+
+```dockerfile
+FROM gcr.io/your-repository/tomcat
+COPY your-application.war ROOT.war
+```
+
+# Securing your application
+For security purpose it is recommend to start the tomcat instance using the tomcat user. 
+
+You can do so by adding the following line at the end of your Dockerfile.
+
+```dockerfile
+RUN chown tomcat:tomcat $CATALINA_BASE/webapps/ROOT.war
+USER tomcat
```

Comment:
Can we just wrap `catalina.sh` and change user there before starting the server?